resources/todomvc/architecture-examples/preact/src/todo/components/item.jsx

```diff
@@ -0,0 +1,51 @@
+import { useState, useCallback } from "preact/hooks";
+import { memo } from "preact/compat";
+
+import { Input } from "./input";
+
+export const Item = memo(function Item({ todo, dispatch }) {
+    const [isWritable, setIsWritable] = useState(false);
+    const { title, completed, id } = todo;
+
+    const toggleItem = useCallback(() => dispatch({ type: "TOGGLE_ITEM", payload: { id } }), [dispatch]);
+    const removeItem = useCallback(() => dispatch({ type: "REMOVE_ITEM", payload: { id } }), [dispatch]);
+    const updateItem = useCallback((id, title) => dispatch({ type: "UPDATE_ITEM", payload: { id, title } }), [dispatch]);
+
+    const handleDoubleClick = useCallback(() => {
+        setIsWritable(true);
+    });
+
+    const handleBlur = useCallback(() => {
+        setIsWritable(false);
+    });
```

Comment:
In these calls, because there's no dependency parameter, my understanding is that `useCallback` will always return the new function, so it's pretty much useless. Probably you wanted to pass an empty array instead, but I'd suggest to just get rid of `useCallback` usage at all here... unless we want to recreate development mistake on purpose (?).

Reply:
will adjust